### PR TITLE
Allow specifying the container runtime to the templated ignition file

### PR DESCRIPTION
**Pull request by kleesc**
Allow specifying the container runtime in the executor's ignition
file. This allow for different runtimes, e.g Docker, Podman to run a build.

**Issue:** https://issues.redhat.com/browse/PROJQUAY-1295

**Changelog:** 

**Docs:** 

**Testing:** 

**Details:**

### CODE REVIEW

File: buildman/templates/dockersystemd.json [hunk 1]
```diff
@@ -37,6 +55,7 @@ ExecStartPost={{ start_post }}
 {% if not oneshot -%}
 ExecStop=/usr/bin/docker stop {{ name }}
 {%- endif %}
```
Comment (thomasmckay) [inline buildman/templates/dockersystemd.json:57]: should this be just `%` instead of `%-`?

Comment (kleesc) [reply buildman/templates/dockersystemd.json:57]: Shouldn't matter. The dash just removes whitespace before that block.

**Review by thomasmckay — APPROVED** [review]
LGTM

**Review by thomasmckay — APPROVED** [review]
LGTM

**Pull request merged**